src/foam/mlang/predicate/CapabilityAuthServicePredicate.js

```diff
@@ -53,7 +54,8 @@ foam.CLASS({
           if ( ucj.getStatus() == CapabilityJunctionStatus.GRANTED ) {
             Capability c = (Capability) getCapabilityDAO().find(ucj.getTargetId());
             if ( c != null && c.getAssociatedEntity().equals(getEntity()) && ! c.isDeprecated(x) ) {
-              c.setX(x);
+              // only set context - which is system - to spid caps - for prerequisiteImplies
+              if ( c instanceof ServiceProvider ) c.setX(x);
```

Comment:
this if is just a precaution - normal capabilities don't need a context - and since its the system I thought it better to not set if not needed